pkg/resource/addon/sdk.go

```diff
@@ -457,17 +457,6 @@ func (rm *resourceManager) sdkUpdate(
 	// the original Kubernetes object we passed to the function
 	ko := desired.ko.DeepCopy()
 
-	if resp.Update.CreatedAt != nil {
-		ko.Status.CreatedAt = &metav1.Time{*resp.Update.CreatedAt}
-	} else {
-		ko.Status.CreatedAt = nil
-	}
-	if resp.Update.Status != nil {
-		ko.Status.Status = resp.Update.Status
-	} else {
-		ko.Status.Status = nil
-	}
-
```

Comment:
Mmmmm is there a reason why we're getting rid of those?

Reply:
This got deleted since I removed `Update` shape in generator.yaml. I explained the reason in other comment. But I did not anticipate , it would delete the shape from `addon` as well. Should I keep `Update`?

Reply:
I updated generator.yaml and added `  - AssociateIdentityProviderConfigOutput.Update`. This will only remove `update` field from identity-provider-config. and wont touch `Addon`